src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-sync-pipe-modal/rgw-multisite-sync-pipe-modal.component.spec.ts

```diff
@@ -89,6 +89,10 @@ describe('RgwMultisiteSyncPipeModalComponent', () => {
     component.submit();
     expect(spy).toHaveBeenCalled();
     expect(putDataSpy).toHaveBeenCalled();
-    expect(putDataSpy).toHaveBeenCalledWith(component.pipeForm.getRawValue());
+    expect(putDataSpy).toHaveBeenCalledWith({
+      ...component.pipeForm.getRawValue(),
+      mode: '',
+      user: ''
```

Comment:
Lets add one test case where "user" and "mode" which verifies that these values are populated correctly and are passed, since its a new addition.

Also one for checking of removal of bucket and that it we are replacing it with "*"

Reply:
can't write the other test as the backend is handling the '*' thing, we are just sending empty value

Reply:
Np! We can add something in our e2e tests for that. Thanks

Reply:
Need to setup multisite configuration in jenkins env, only after that we can test flows and pipe.
Thanks